### ISSUE TITLE
Force refresh

### ISSUE THREAD

**Issue by Akibalogh** [chasdevs/meetrics#4]
One more question: I understand this script runs overnight. Is there a command to force it to refresh immediately?

**Comment by Akibalogh** [comment]
I see the function computeYesterday() so I am guessing it's only designed to run overnight

**Comment by chasdevs** [comment]
Yup! I had a cron running every day and it would just compute yesterday. _backfill.go_ is the best way to compute a whole range of dates; I didn't go any more granular than daily (no hourly refreshing at the moment).

**Comment by Akibalogh** [comment]
Thanks.. I was just thinking of whether I could set my system clock to a different timezone, or if there's a docker setting.. will look deeper this wknd
